### ISSUE TITLE
application.properties template for Docker

### ISSUE THREAD

**Issue by BlackRose01** [eclipse-arrowhead/core-java-spring#289]
Hello,

currently I try to run Arrowhead on Docker via CI/CD but my problem is that all given application.properties are filled with example entries. This is counterproductive because the effort to automate the process is very high.

Therefore, would it be possible that in one of the Docker folders of the repository all application.properties files could be stored with placeholders? This makes it far easier to run a bash script or similar over them, which then replaces them afterwards.

A small idea: create the placeholders in a special pattern. Thus, for example, the JDBC database connection in the Orchestrator could be abbreviated with the placeholder "ORCH_JDBC_URL".

KR,
BlackRose01

**Comment by tsvetlin** [comment]
The docker containers support mounting the application properties as an external volume.
See the docker compose file for an example.
https://github.com/eclipse-arrowhead/core-java-spring/blob/master/docker/docker-compose.yml

Currently what you mean under "placeholder" are the default values, so people have an idea what to write in there, moreover they have the ability to change them.

**Comment by BlackRose01** [comment]
Thanks for the information but the problem is that the last update of the Docker images was at least a year ago and I don't see which version of Arrowhead is used. So the purpose for testing new deployments is not there.

I also tried replacing the application.properties all with configuration files that have the placeholders. Unfortunately this did not work as I had hoped. For this reason I looked for another solution and found it. I wrote the concept in the new ticket #303.
